package.json

```diff
@@ -45,7 +45,6 @@
     "ember-faker": "1.1.0",
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
-    "ember-get-helper": "1.0.4",
```

Comment:
This will probably break the 1.13 tests. Ideally, we would add this to the consuming app's package.json in the default blueprint if < Ember 2.0.

For now we should at least add it to the 1.13 ember-try scenario